### DISCUSSION TITLE
Technology review

### DISCUSSION THREAD

**Discussion by infinite0x20** [anhourandaquarter/carmina#8] [General]
In preparation for our technology review presentation on Tuesday, I figured we can start a thread here.

Topics:
- Classical Latin (`cltk`)
- String parsing
- Text analysis
- Web parsing (`selenium`, `requests`, `beautifulSoup`) -- for getting to Logeion if we want to use it to verify Latin text.

I can look into `cltk`. Let's look at some other packages for string parsing and/or text analysis. I can think of `nltk`, `TextBlob`, and `spaCy`, though there might be others.

**Comment by infinite0x20** [comment]
Hey y'all, just made some Google Slides for us to work on over the long weekend. They're pretty bare-bones, but we can always fix style/theme stuff later.

**Comment by thatswhatsimonsaid** [comment]
Got it! I'll be on it
